netket/nn/linear.py

```diff
@@ -207,6 +208,99 @@ def __call__(self, inputs: Array) -> Array:
         return y
 
 
+class DenseSymm(Module):
+    permutations: Any
+    alpha: Union[float, int]
```

Comment:
I'd switch the two around, and check if my callable signature is ok
```suggestion
    alpha: Union[float, int]
    permutations: Callable[[], Array]
```

Reply:
also, flax uses features by default (which is not alpha).
In this case our Alpha is features, no?

Reply:
Hm, I am still a bit confused by the `alpha`, `features` terminology (at least in the case where `N_sym != N_sites`). Since `alpha` is used only to compute `self.features`, it should be possible to just pass features - we just need to make sure the code works e.g. for non-integer `alpha`s (not sure if it actually does right now).

Reply:
if` N_sym == N_sites ` we all agree that `alpha=features` in this case.

In the other case... alpha doesn't make too much sense, but features does.
At least in my mind.

You see it because the `int` call down there fore features is flooring the result of the multiplication, which I think might lead to subtle bugs and I'd avoid, even if that was our old behaviour.

Reply:
cc @gcarleo

Reply:
The `DenseSymm` layer uses `features` now, with `n_hidden = features * n_symm`. `RBMSymm` still has `alpha` for compatibility with `RBM`.